src/filters/pretty-bytes-filter.js

```diff
@@ -0,0 +1,28 @@
+// usage: {{ file.size | prettyBytes }}
```

Comment:
Had to add this as a custom filter, as none of the packages worked.

Reply:
nice.